nixos/tests/minidlna.nix

```diff
@@ -0,0 +1,39 @@
+import ./make-test.nix ({ pkgs, ... }: {
+  name = "minidlna";
+
+  nodes = {
+    server =
+      { ... }:
+      {
+        imports = [ ../modules/profiles/minimal.nix ];
+        networking.firewall.allowedTCPPorts = [ 8200 ];
+        services.minidlna = {
+          enable = true;
+  	  loglevel = "error";
+  	  mediaDirs = [
+            "PV,/tmp/stuff"
+          ];
+          friendlyName = "rpi3";
+          rootContainer = "B";
+          extraConfig =
+	  ''
+album_art_names=Cover.jpg/cover.jpg/AlbumArtSmall.jpg/albumartsmall.jpg
+album_art_names=AlbumArt.jpg/albumart.jpg/Album.jpg/album.jpg
+album_art_names=Folder.jpg/folder.jpg/Thumb.jpg/thumb.jpg
+notify_interval=60
+          '';
+        };
+      };
+    client = { ... }: { };
```

Comment:
Why is this declared? Isn't being used.

Reply:
because i made an error in the test, i opened the port 8200 to check i can curl the server on it.
and i called that test with $server instead...

Thanks for pointing it out.

Reply:
Fixed by doing initially what i aimed at as explained.
Amended the last commit and pushed.

Reply:
I still don't see client being used... ?